_posts/2020-08-19-cockpit-226-cockpit-podman-22.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Cockpit 226 and Cockpit Podman 22
+author: mvo
+date: 2020-08-19
+tags: cockpit linux
+slug: cockpit-226
+category: release
+summary: Cockpit and Cockpit Podman news
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).
+We release regularly. Here are the release notes from Cockpit version 226 and Cockpit Podman version 22.
+
+### Storage: Better support for "noauto" LUKS devices
+
+Cockpit now behaves more sensible with encrypted filesystems that have
+"noauto" in their LUKS options. Such a filesystem will now also always
+have "noauto" in their mounting options even when mounted.
+
+### Podman: Support for pod group deletion
+
+The cockpit-podman plugin gained the ability to delete pod groups.
+
+![Pod group deletion](/images/podman-pod-deletion.png){:.screenshot}
```

Comment:
```suggestion
![Pod group deletion](/images/podman-pod-deletion.png)
```

We don't necessarily need the screenshot class on a screenshot. It's intended to add a border around partial screenshots. This screenshot already has a border inside the image itself.

(Perhaps I should rename the class.)

Reply:
Ah, I see! I just copied this from the previous release notes without understanding what it does.